src/services/movies.services.ts

```diff
@@ -0,0 +1,91 @@
+import { omit } from 'lodash';
+
+import { CustomError, Movie } from '../interfaces';
+import { GenreModel, MovieModel } from '../models';
+
+export async function getMovies() {
+    try {
+        const movies = await MovieModel.find();
+        return omit(movies, 'result');
```

Comment:
Why do we need omit from lodash in your functions?
It accepts objects, not array

Reply:
Actually, we don't. 
I have used it before in one of the courses. Here I just copy/paste it and did not even think for what. 
Thanks to your comment I revised it.
Now it looks more correct.
Thank you very much!

Reply:
![image](https://github.com/sancho-sumy/Node.js_Course_Practice/assets/51134737/0757ee39-6ac4-4cc3-be2d-50e145a7755d)
You can also use the second paramater of mongoose methods to exclude some fields from response

Reply:
Replaced `omit` by adding a second parameter to the MongoDB method.